rpm/screen.spec

```diff
@@ -32,20 +31,12 @@ Summary:    Documentation for %{name}
 Requires:   %{name} = %{version}-%{release}
 Requires(post):   /sbin/install-info
```

Comment:
Do we need this here? We have been dropping this from all packages in general.

Reply:
Well, if we keep the info file then this should be called and guess there was no file trigger added for this. 

Alternatively guess we could skip the info file side and just have the man page in the doc package. Suites me.